compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -620,9 +620,23 @@ bool OMR::ARM64::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
    if(length != TR::VectorLength128) return false;
 
    // implemented vector opcodes
+
+   if (opcode.isVectorOpCode() && opcode.getVectorOperation() == OMR::vadd)
+      {
+      TR::DataType ot = opcode.getVectorResultDataType();
+
+      if (ot.getVectorLength() != TR::VectorLength128) return false;
+
+      TR::DataType et = ot.getVectorElementType();
+
+      if (et == TR::Int8 || et == TR::Int16 || et == TR::Int32 || et == TR::Int64 || et == TR::Float || et == TR::Double)
```

Comment:
The same comment as in https://github.com/eclipse/omr/pull/6458/files#r843332926. It appears that all element types are listed here.

Reply:
Right, I just copied the old code. I will simplify.

Reply:
done